### PR TITLE
Reduce indentation on packages installed for Drydock

### PR DESCRIPTION
**Pull request by wilkers-steve**
Indentation was excessive

---
This change is [Reviewable](https://reviewable.io/reviews/att-comdev/dockerfiles/103)

### CODE REVIEW

File: drydock/Dockerfile [hunk 1]
```diff
@@ -20,17 +20,18 @@ ENV DEBIAN_FRONTEND=noninteractive \
...
-                         gcc \
-                         git \
-                         libffi-dev \
-                         libssl-dev --no-install-recommends \
-                         make \
-                         netbase \
-                         python3-minimal \
-                         python3-setuptools \
-                         python3-pip python3-dev
+    apt-get -y install \
```
Comment (intlabs) [inline drydock/Dockerfile]: could you add `--no-install-recommends` to reduce excess bloat?

File: drydock/Dockerfile [hunk 2]
```diff
@@ -20,17 +20,18 @@ ENV DEBIAN_FRONTEND=noninteractive \
...
+       g++ \
+       gcc \
+       git \
+       libffi-dev \
+       libssl-dev --no-install-recommends \
+       make \
+       netbase \
+       python3-minimal \
+       python3-setuptools \
+       python3-pip python3-dev
```
Comment (intlabs) [inline drydock/Dockerfile]: could you put one package per line?

File: drydock/Dockerfile [hunk 3]
```diff
@@ -20,17 +20,18 @@ ENV DEBIAN_FRONTEND=noninteractive \
...
```
Comment (intlabs) [inline drydock/Dockerfile]: could you add `rm -rf /var/lib/apt/lists/*` to remove the apt cache?

**Review by intlabs — CHANGES_REQUESTED** [review]
Looks good, just a couple of comments.

**Comment by wilkers-steve** [comment]
@intlabs Thanks Pete.  Changes made

**Comment by intlabs** [comment]
LGTM :) Thanks Steve

**Pull request merged**